channel-config/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,7 @@
+[[extensions]]
+type = "channel_config"
```

Comment:
I believe `name = ` is also a required property if I'm not mistaken, so you can add that as well